src/middleware/secure-headers/index.ts

```diff
@@ -129,15 +171,49 @@ function getFilteredHeaders(options: SecureHeadersOptions): [string, string][] {
 
 function getCSPDirectives(
   contentSecurityPolicy: SecureHeadersOptions['contentSecurityPolicy']
```

Comment:
This can be:

```ts
contentSecurityPolicy: ContentSecurityPolicyOptions
```

Reply:
Thank you!

As this is code that predates this PR and is consistent with other parts of the current code, I would like to leave it unchanged.

https://github.com/honojs/hono/blob/06fe3ecd336d6db8eec69ca7796d2720019f6ba8/src/middleware/secure-headers/index.ts#L144

Reply:
Oh, sorry, it's related to the comment below that one. Let's change it!

Reply:
fixed in [a6e1277](https://github.com/honojs/hono/pull/2577/commits/a6e1277e949deaf6aebe1fa516a14644aa69dbab)